k8s/production/gitlab/release.yaml

```diff
@@ -101,6 +103,10 @@ spec:
       appConfig:
         smartcard:
           enabled: false
+        lfs:
+          enabled: false
+        packages:
+          enabled: false
```

Comment:
We don't use these services, and disabling them lets us avoid creating S3 buckets for them.

Reply:
(@zackgalbreath or @scottwittenburg can you confirm that we don't use these?)

Reply:
That is my understanding. We don't host packages at gitlab.spack.io, and spack's GitHub repository doesn't use git lfs, so there's no need for it on the gitlab mirror.